.gitignore

```diff
@@ -1,33 +1,3 @@
-# Logs
-logs
-*.log
-npm-debug.log*
-
-# Runtime data
-pids
-*.pid
-*.seed
-
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
-# Coverage directory used by tools like istanbul
-coverage
-
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
-
-# node-waf configuration
-.lock-wscript
-
-# Compiled binary addons (http://nodejs.org/api/addons.html)
-build/Release
-
-# Dependency directory
 node_modules
```

Comment:
I trimmed down the `.gitignore` file to just the things needed for this repo and I deleted the `.npmignore` file as `npm` defaults to using the `.gitignore` file if there isn't a `.npmignore` file present

> "If there's no .npmignore file, but there is a .gitignore file, then npm will ignore the stuff matched by the .gitignore file." -- [Keeping files out of your package](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package)